lms/static/sass/views/_login-register.scss

```diff
@@ -3,22 +3,81 @@
 @import '../base/grid-settings';
 @import "neat/neat"; // lib - Neat
 
-.login-register {
+@media (min-width: 768px) {
+    .enterprise-content {
+        width: 20%;
+        float: left;
+        height: 100%;
+    }
+
+    .border-left {
+        border-left: 1px solid #d9d9d9;
+    }
+}
+
+@media (max-width: 767px) {
+    .enterprise-content {
+        margin: auto auto;
+        display: block;
+    }
+
+    img {
+        display: none;
```

Comment:
This is hiding all of the images on the page (including the enterprise logo) at mobile width, but the testing instructions say:
> Verify that if you resize the window down to be very narrow, the Enterprise Customer logo moves to be on top of the registration form.

Should this be fixed so that the enterprise logo is displayed?

![](https://i.imgur.com/QWJrA1G.png)